.coveragerc

```diff
@@ -510,6 +511,179 @@ omit =
     homeassistant/components/sesame/lock.py
     homeassistant/components/seven_segments/image_processing.py
     homeassistant/components/seventeentrack/sensor.py
+    homeassistant/components/sensor/aftership.py
```

Comment:
Some unrelated changes sneaked in.

Reply:
These changes in `.coveragerc`.

Reply:
Try rebasing on lastest dev branch.

Reply:
Something is foo'ed :(

`First, rewinding head to replay your work on top of it...
Applying: Added Nordpool Sensor platform
Applying: Added coveragerc and requirements
Using index info to reconstruct a base tree...
M       .coveragerc
M       requirements_all.txt
Falling back to patching base and 3-way merge...
Auto-merging requirements_all.txt
CONFLICT (content): Merge conflict in requirements_all.txt
Auto-merging .coveragerc
CONFLICT (content): Merge conflict in .coveragerc
error: Failed to merge in the changes.
Patch failed at 0002 Added coveragerc and requirements
The copy of the patch that failed is found in: .git/rebase-apply/patch
`

Reply:
Think I got the files reset to dev and added requirements and coverage for this sensor.